Tests/JSONs/segment-field.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "id": "group",
+    "title": "Group",
+    "sections": [
+      {
+        "id": "section",
+        "fields": [
+          {
+            "id": "segment",
+            "title": "My Segment",
+            "type": "segment",
+            "value": "part_1",
```

Comment:
This wasn't needed. :)

https://github.com/hyperoslo/Form/pull/545/commits/e56a5bd6aa7140ce5751e290255a0c05df0cb2f3